components/drawer/DrawerContent.js

```diff
@@ -43,11 +54,18 @@ const DrawerContent = (props) => {
         <Text>Logo here</Text>
       </View>
       <View style={styles.bottomContainer}>
-        <RNButton title="Using" onPress={() => Actions.using()} />
-        <RNButton title="Responding" onPress={() => Actions.responding()} />
-        <RNButton title="User Profile" onPress={() => Actions.profile()} />
-        <RNButton title="Resource" onPress={() => Actions.resource()} />
-        <RNButton title="Logout" onPress={() => setModalVisible(true)} />
+        <List
```

Comment:
I thought we had a List and ListItem component ourselves. Did you try to use that one?